test/WebJobs.Script.Tests/Configuration/FunctionsHostingConfigOptionsTest.cs

```diff
@@ -16,6 +16,28 @@ namespace Microsoft.Azure.WebJobs.Script.Tests.Configuration
 {
     public class FunctionsHostingConfigOptionsTest
     {
+        [Fact]
```

Comment:
use `Theory` instead and add tests for different casing - all upper, all lower, mix of lower and upper case